### PR TITLE
OSMTemplates for buildings, landcover and water 

### PR DESCRIPTION
**Pull request by ELSW56**
The processes to extract buildings, landcover and water have been implemented.

### CODE REVIEW

File: osm/src/main/groovy/org/orbisgis/osm/Loader.groovy [hunk 1]
```diff
@@ -84,7 +84,7 @@ boolean executeOverPassQuery( def query,  def outputOSMFile) {
     if (outputOSMFile.exists()) {
         outputOSMFile.delete()
     }
-    def apiUrl = "https://overpass-api.de/api/interpreter?data="
+    def apiUrl = "https://lz4.overpass-api.de/api/interpreter?data="
```
Comment (ebocher) [inline osm/src/main/groovy/org/orbisgis/osm/Loader.groovy]: use official overpass api

File: osm/src/main/groovy/org/orbisgis/osm/OSMTemplate.groovy [hunk 2]
```diff
@@ -51,6 +105,59 @@ import org.orbisgis.processmanagerapi.IProcess
      * @return
      * @author Erwan Bocher
```
Comment (ebocher) [inline osm/src/main/groovy/org/orbisgis/osm/OSMTemplate.groovy:108]: add author Le Saux

File: osm/src/main/groovy/org/orbisgis/osm/OSMTemplate.groovy [hunk 3]
```diff
@@ -39,10 +64,39 @@ import org.orbisgis.processmanagerapi.IProcess
...
      */
-    IProcess LANDCOVER() {
-        // keys = 'landcover', 'natural', 'landuse', 'water', 'waterway', 'leisure', 'aeroway', 'amenity'
-
-    }
+public IProcess LANDCOVER() {
```
Comment (ebocher) [inline osm/src/main/groovy/org/orbisgis/osm/OSMTemplate.groovy:69]: add author Le Saux

**Review by ebocher — APPROVED** [review]

File: osm/src/main/groovy/org/orbisgis/osm/OSMTemplate.groovy (hunk 3 above)
Comment (ELSW56) [reply osm/src/main/groovy/org/orbisgis/osm/OSMTemplate.groovy:69]: Done

File: osm/src/main/groovy/org/orbisgis/osm/OSMTemplate.groovy (hunk 2 above)
Comment (ELSW56) [reply osm/src/main/groovy/org/orbisgis/osm/OSMTemplate.groovy:108]: Done

**Pull request merged**